website/content/en/releases/14.3R/hardware.adoc

```diff
@@ -2528,6 +2528,17 @@ sysctl(8).
 
 
 
+The
+**ure**
+device driver provides support for the following Ethernet controllers:
+
+*	Realtek RTL8156/RTL8156B/RTL8156BG
+*	Realtek RTL8153/RTL8153B
+*	Realtek RTL8152
+*	Realtek RTL8168/8169/8110/8211
+
+
```

Comment:
You shouldn't type this manually, you can use the same macro as you did in the file above, and then it will get regenerated several times in the process of the release. I think. Cc @cperciva

Reply:
For now all we need is the change to archetypes; I'll regenerate the 14.3 hardware notes from there when I create releng/14.3.

But adding @kenrap's text here is fine and will have it showing up on the website immediately rather than waiting for the next regeneration.

Reply:
It's now been changed to `&hwlist.ure;` for simplicity and sync-ability.